modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/helper/AddFormInstanceRecordMVCCommandHelper.java

```diff
@@ -328,6 +351,10 @@ private void _removeValue(
 	@Reference
 	private DDMFormEvaluator _ddmFormEvaluator;
 
+	@Reference
+	private DDMFormInstanceRecordVersionLocalService
```

Comment:
I understand that this is an OSGi component and there's already a case of referencing a service that it is also used in the command, but what do you think of not making this reference and just carrying as a parameter the service from the commands that call this method? I don't think we need to make sure in the helper that the service is available since the commands are already components which are the only users of the helper and that make sure of that.
Does it make sense? Let me know your thoughts.

Reply:
It makes sense! Thanks!

Reply:
Done!